api/audit/views.py

```diff
@@ -44,18 +48,42 @@ def get_queryset(self) -> QuerySet[AuditLog]:
         if search:
             q = q & Q(log__icontains=search)
 
-        return AuditLog.objects.filter(q).select_related(
+        queryset = AuditLog.objects.filter(q).select_related(
             "project", "environment", "author"
         )
 
-    def _get_base_filters(self) -> Q:
-        return Q()
+        return self._apply_visibility_limits(queryset)
 
     def get_serializer_class(self):
         return {"retrieve": AuditLogRetrieveSerializer}.get(
             self.action, AuditLogListSerializer
         )
 
+    def _get_base_filters(self) -> Q:  # pragma: no cover
```

Comment:
This is essentially just an abstract method. I did think about whether I should mark it as an abstract method and/or use `NotImplementedError()` but I decided that it was fine to just keep it as it is since there could in the future be a use case where we don't need to override this method.